src/infra/marketplace/marketplaceQueryRepo.go

```diff
@@ -355,6 +355,37 @@ func (repo *MarketplaceQueryRepo) parseCatalogItemCmdSteps(
 	return itemCmdSteps, nil
 }
 
+func (repo *MarketplaceQueryRepo) parseCatalogItemUninstallFileNames(
+	catalogItemUninstallFileNames interface{},
+) (itemUninstallFileNames []valueObject.UnixFileName, err error) {
+	if catalogItemUninstallFileNames == nil {
```

Comment:
How this would be nil if you checked on line 511 if it's not nil?

Reply:
I removed the validation. I also took the opportunity to remove it from other methods that had the same issue.